Dockerfile.dapper

```diff
@@ -27,7 +27,7 @@ RUN git clone https://github.com/sstephenson/bats.git && \
 ENV DAPPER_SOURCE /go/src/github.com/rancher/rancher-dns/
 ENV DAPPER_OUTPUT ./bin ./dist
 ENV DAPPER_DOCKER_SOCKET true
-ENV DAPPER_ENV REPO TAG
+ENV DAPPER_ENV REPO TAG CROSS WINDOWS_DOCKER_HOST
```

Comment:
Where does `WINDOWS_DOCKER_HOST` get evaluated?
Where is `CROSS` set?

Reply:
When they are releasing micro-services, they will set CROSS=1 in env var. already confirmed with Craig and Denise.